3.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tailwind</title>
+</head>
+<body>
+    <header class="header">
+        <div class="header__left-column">
+            <h2 class="header__left-column__h2">Press</h2>
+            <p class="header__left-column__text">Et malesuada fames ac turpis egestas sed tempus. </p>
+        </div>
+        <div class="header__right-column">
+            <form action="GET" class="header__right-column form">
+                <input type="text" placeholder="Enter your name" class="form__input">
+                <button type="submit" class="form__button">Notify me</button>
+            </form>
+        </div>
+    </header>
+    <div class="main-content">
+        <article class="main-content__element">
+            <time class="dtstart" datetime="2005-10-05" class="main-content__element__time">October 5</time>
+            <h3 class="main-content__element__header">title1</h3>
+            <p class="main-content__element__text">unc consequat interdum varius. Sed risus ultricies tristique nulla aliquet. Massa enim nec dui nunc mattis. Vitae tortor condimentum lacinia quis. Scelerisque eleifend donec pretium vulputate sapien. Tortor dignissim convallis aenean et tortor at risus viverra adipiscing. A condimentum vitae sapien pellentesque.</p>
+            <footer  class="main-content__element__footer">
+                <a href="" class="footer__text">Read full story</a>
+            </footer>
+        </article>
```

Comment:
```suggestion
        <article class="main-content__element">
            <time datetime="2005-10-05" class="main-content__time">October 5</time>
            <h3 class="main-content__header">title1</h3>
            <p class="main-content__text">unc consequat interdum varius. Sed risus ultricies tristique nulla aliquet. Massa enim nec dui nunc mattis. Vitae tortor condimentum lacinia quis. Scelerisque eleifend donec pretium vulputate sapien. Tortor dignissim convallis aenean et tortor at risus viverra adipiscing. A condimentum vitae sapien pellentesque.</p>
            <footer  class="main-content__footer">
                <a href="" class="main-content__footer-text">Read full story</a>
            </footer>
        </article>
```

или карточки новости придумать свой блок (например .card), а в внутри описать все элементы блока card